### PR TITLE
Add Chinese translations for some GUIs

### PR DESCRIPTION
**Pull request by yangqing8230**
Add Chinese translations for AddBookmarkDialog / AddTLESourceDialog /AfcControl / AskControl and update ColorConfigTab.

### CODE REVIEW

File: i18n/SigDigger_zh_CN.ts [hunk 1]
```diff
@@ -2349,5 +2349,219 @@
...
+    <message>
+        <location filename="../ui/AddBookmarkDialog.ui" line="71"/>
+        <source>Modulation</source>
+        <translation type="finished">调制</translation>
+    </message>
+    <message>
+        <location filename="../ui/AddBookmarkDialog.ui" line="94"/>
+        <location filename="../ui/AddBookmarkDialog.ui" line="98"/>
+        <source>AM</source>
+        <translation type="finished">不可用</translation>
```
Comment (BatchDrake) [inline i18n/SigDigger_zh_CN.ts:2383]: Why have you translated AM as 不可用?

File: i18n/SigDigger_zh_CN.ts [hunk 2]
```diff
@@ -2349,5 +2349,219 @@
...
+    </message>
+    <message>
+        <location filename="../ui/AddBookmarkDialog.ui" line="103"/>
+        <source>FM</source>
+        <translation type="unfinished"></translation>
```
Comment (BatchDrake) [inline i18n/SigDigger_zh_CN.ts:2388]: You didn't translate FM, but you did translate USB, LSB, RAW and perhaps AM. Is this intentional?

**Comment by BatchDrake** [comment]
Hi Yáng, 

Thanks for your PR. Could you address these two comments I added earlier? As soon as I have your changes I will merge ASAP.

Cheers,

**Comment by BatchDrake** [comment]
Oh, btw, in this PR you are saying that you are adding Chinese translations for AddTLESourceDialog /AfcControl / AskControl, but I only saw the translations for AddTLESourceDialog. Are you planning to add more changes to this PR?

File: i18n/SigDigger_zh_CN.ts (hunk 1 above)
Comment (yangqing8230) [reply i18n/SigDigger_zh_CN.ts]: This is a mistake. I correct it.

File: i18n/SigDigger_zh_CN.ts (hunk 2 above)
Comment (yangqing8230) [reply i18n/SigDigger_zh_CN.ts]: Yes, as you metioned, I intentionnally kept AM/FM untranslated before, due to these term are quit familiar to us.
But when I looked at your question for a while, I changed my mind. I translated AM and FM to Chinese terms "调幅" and "调频", reasons as below:
1.  These Chinese terms are even more common and acceptable.
2. To keep the GUI appearance coherent.

**Comment by yangqing8230** [comment]
I really added them in this PR. The context of AFCControl begins at line 2433. Is there something mistaken in my git operation?

**Comment by BatchDrake** [comment]
My bad, GitHub UI played tricks on me. Everything is fine, merging now.

**Pull request merged**